src/ceph-volume/plugin/zfs/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+
+include(Distutils)
+distutils_install_module(ceph_volume_zfs
```

Comment:
is `ceph_volume_zfs` supposed to be a separated python module which depends on `ceph-volume` or a plugin of `ceph-volume`?

Reply:
@tchaikov @alfredodeza 
I'm trying to redo a little of the things as possible, and reuse most of ceph-volume.
So that would make it a full part of ceph-volume, and as it is now it will not function without ceph-volume also being present.